cmd/keytransparency-server/frontend.go

```diff
@@ -202,7 +202,7 @@ func main() {
 	mutator := entry.New()
 
 	// Connect to log server.
-	tconn, err := grpc.Dial(*logURL)
+	tconn, err := grpc.Dial(*logURL, grpc.WithInsecure())
```

Comment:
When Trillian adds TLS support, we can modify our code to enforce it, but that should be a later PR.